### PR TITLE
migration84/new-features.xml: Add description and example for lazy objects

### PR DESCRIPTION
**Pull request by arnaud-lb**

### CODE REVIEW

**Review by jimwins — APPROVED** [review]
LGTM

File: appendices/migration84/new-features.xml [hunk 1]
```diff
@@ -55,10 +55,34 @@ class Example
   <!-- RFC: https://wiki.php.net/rfc/lazy-objects -->
   <sect3 xml:id="migration84.new-features.core.lazy-objects">
    <title>Lazy Objects</title>
-
    <simpara>
-    TODO
+    It is now possible to create objects whose initialization is deferred until they are accessed. Libraries and frameworks can leverage these lazy objects to defer fetching data or dependencies required for initialization.
```
Comment (Girgias) [inline appendices/migration84/new-features.xml:59]: We have a recommended 80 char per line style guide.

```suggestion
    It is now possible to create objects whose initialization is deferred until they are accessed.
    Libraries and frameworks can leverage these lazy objects to defer fetching data or dependencies required for initialization.
```

File: appendices/migration84/new-features.xml [hunk 2]
```diff
@@ -55,10 +55,34 @@ class Example
...
    </simpara>
+   <informalexample>
+   <programlisting role="php">
```
Comment (Girgias) [inline appendices/migration84/new-features.xml:62]: Nit: indentation seems to be slightly off here in that the programlisting should have an extra indent.

**Review by Girgias — APPROVED** [review]
Minor nits, but LGTM

**Comment by Girgias** [comment]
Thank you!

**Pull request merged**